supervisor/resolution/checks/base.py

```diff
@@ -41,11 +43,16 @@ async def __call__(self) -> None:
             self.sys_resolution.dismiss_issue(issue)
 
         # System is not affected
-        if affected and not self.multiple:
+        if affected and self.context not in (ContextType.ADDON, ContextType.PLUGIN):
             return
         _LOGGER.info("Run check for %s/%s", self.issue, self.context)
         await self.run_check()
 
+    @property
+    def slug(self) -> str:
+        """Return the check slug."""
+        return Path(sys.modules[self.__class__.__module__].__file__).stem
```

Comment:
Wouldn't this be the same:

```suggestion
        return self.__class__.__module__.split('.')[-1]
```

Reply:
We need the file name, not the module folder

Reply:
`stem` returns the filename without extension. That is the same as the last part of the module.

![image](https://user-images.githubusercontent.com/1444314/110900026-f752f580-82b6-11eb-9a82-f3844ed78822.png)

Reply:
🤷 let see, we have tests for that